src/AppBundle/Entity/Client.php

```diff
@@ -554,6 +554,19 @@ public function getUnsubmittedReports()
     }
 
     /**
+     * @JMS\VirtualProperty
+     * @JMS\Type("AppBundle\Entity\Report\Report")
+     * @JMS\SerializedName("current_report")
+     * @JMS\Groups({"current-report"})
+     *
+     * @return Report|null
+     */
+    public function getCurrentReport()
+    {
+        return $this->getUnsubmittedReports()->first() ?: null;
```

Comment:
If the order of reports in the method above is altered then this will start to fail. Can we put through a sort function to ensure that the current report always returns the current report?